Solutions/Darktrace/Data Connectors/Doc_DarktraceConnectorRESTAPI.md

```diff
@@ -0,0 +1,39 @@
+<!-- Taken from https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/Templates/Doc_Template_REST_API_Connector.md -->
+
+# Connect your Darktrace Enteprise Immune System to Microsoft Sentinel 
```

Comment:
@JustasZDT, please fix the typo Enteprise to Enterprise

Reply:
Typo has been fixed now.